### PR TITLE
Work history being entered from before applicants graduated/qualified

### PR DESCRIPTION
**Pull request by David-Feetenby**
Add content and banner logic to help stop unneeded work history

### CODE REVIEW

File: spec/system/teacher_interface/check_answers_spec.rb [hunk 1]
```diff
@@ -67,6 +67,70 @@
...
+      )
+    end
+  end
+
+  it "should not display the banner if both earliest_certificate_date and earliest_work_history_date are blank" do
+    @application_form.qualifications = []
+    @application_form.work_histories = []
+
+    #reload the page
+    visit current_path
```
[the remark this answers is not in the document]
Comment (David-Feetenby) [reply spec/system/teacher_interface/check_answers_spec.rb]: The test seems to fail if i don't specify it. Not sure why, but currently in the process of refactoring them anyway so hopefully will just get superceded.

File: app/helpers/application_form_helper.rb [hunk 2]
```diff
@@ -108,4 +108,14 @@ def application_form_summary_rows(
       { key: { text: key }, value: { text: value }, actions: actions || [] }
     end
   end
+
+  def display_banner?(application_form)
```
Comment (thomasleese) [inline app/helpers/application_form_helper.rb:112]: Could we clarify the name of this? Something that explains which banner is being displayed would be ideal.

File: spec/system/teacher_interface/check_answers_spec.rb [hunk 3]
```diff
@@ -67,10 +67,74 @@
     end
   end
 
+  it "display banner" do
```
Comment (thomasleese) [inline spec/system/teacher_interface/check_answers_spec.rb:70]: Would be good to explain which banner we're testing here, but the test itself looks good!

File: app/helpers/application_form_helper.rb [hunk 4]
```diff
@@ -108,4 +108,14 @@ def application_form_summary_rows(
       { key: { text: key }, value: { text: value }, actions: actions || [] }
     end
   end
+
+  def display_work_history_banner?(application_form)
```
Comment (thomasleese) [inline app/helpers/application_form_helper.rb:112]:
```suggestion
  def application_form_display_work_history_before_teaching_qualification_banner?(application_form)
```

The helpers are global methods across all the views, so for a convention we've prefixed them with the model they're related to `application_form`.

**Review by thomasleese — APPROVED** [review]

**Pull request merged**